component/vdr/mock/mock_vdr.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package mock
+
+import (
+	"github.com/hyperledger/aries-framework-go/component/models/did"
+	vdrapi "github.com/hyperledger/aries-framework-go/spi/vdr"
+)
+
+// MockVDR mock implementation of vdr
+// to be used only for unit tests.
+type MockVDR struct { // nolint: golint // ignore stutter
```

Comment:
Similar comment as above - can be changed to simply `VDR` to avoid the stutter

Reply:
Done!